src/components/Table/Table.tsx

```diff
@@ -66,7 +68,7 @@ type SortType = {
 const mapToTableAriaProps = (columns: Column[], rows: Row[]): TableStateProps<any> => ({
     children: [
         <TableHeader key="table-header" columns={columns}>
-            {(column) => <AriaColumn allowsSorting>{column.name}</AriaColumn>}
+            {(column) => <AriaColumn allowsSorting={column.sortable}>{column.name}</AriaColumn>}
```

Comment:
Since the sorting function has to be to set by the component user, wouldn't it be better to only show the sorting arrow and/or allow sorting if the `onSortChange` callback is also provided?

Reply:
Yes, it's being done already on `<TableColumnHeader>`